addon/components/o-s-s/power-select.hbs

```diff
@@ -12,34 +17,55 @@
           </span>
         {{/each}}
       </div>
-        <OSS::Icon @icon={{if this.displaySelect "fa-chevron-up" "fa-chevron-down"}}
-            class="dropdown-icon" />
+      <OSS::Icon @icon={{if this.isOpen "fa-chevron-up" "fa-chevron-down"}} class="dropdown-icon" />
     </div>
   </div>
 
-  {{#if this.displaySelect}}
-    {{#if (has-block "empty-state")}}
-      <OSS::InfiniteSelect @items={{@items}} @onSearch={{@onSearch}} @inline={{false}} @onSelect={{this.onSelect}}
-                           @searchPlaceholder={{@searchPlaceholder}} @loading={{@loading}} @loadingMore={{@loadingMore}}
-                           @onBottomReached={{@onBottomReached}}
-                           {{on-click-outside this.hideSelect}}>
-        <:option as |item|>
-          {{yield item to="option-item"}}
-        </:option>
+  {{#if this.isOpen}}
+    {{#in-element this.portalTarget insertBefore=null}}
+      {{#if (has-block "empty-state")}}
+        <OSS::InfiniteSelect
+          @items={{@items}}
+          @onSearch={{@onSearch}}
+          @inline={{false}}
+          @onSelect={{this.onSelect}}
+          @searchPlaceholder={{@searchPlaceholder}}
+          @loading={{@loading}}
+          @loadingMore={{@loadingMore}}
+          @onBottomReached={{@onBottomReached}}
+          class={{concat "margin-top-px-0 upf-power-select__dropdown " this.dropdownAddressableClass}}
```

Comment:
nit:
```
class="magin-top-px-0 upf-power-select__dropdown {{this.dropdownAddressableClass}}"
```

Reply:
less verbose

Reply:
if find it less visible that's why i'm not doing it like that anymore 😅